contracts/core/lib/auction-price-libraries/LinearAuctionPriceCurve.sol

```diff
@@ -28,27 +28,66 @@ import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
 contract LinearAuctionPriceCurve {
     using SafeMath for uint256;
 
+    uint256 public priceDenominator;
+
+    /*
+     * Declare price denominator (or precision) of price curve
+     *
+     * @param  _priceDenominator        The priceDenominator you want this library to always return
+     */
+    constructor(
+        uint256 _priceDenominator
+    )
+        public
+    {
+        // Set price to be returned by library
+        priceDenominator = _priceDenominator;
+    }
+
+    /*
+     * Validate any auction parameters that have library-specific restrictions
+     *
+     * @param  -- Unused auction time to pivot param to conform to IAuctionPriceCurve --
+     * @param  -- Unused auction start price to conform to IAuctionPriceCurve --
+     * @param  _auctionPivotPrice         The price at which auction curve changes from linear to exponential
+     */
+
+    function validateAuctionPriceParameters(
+        uint256,
+        uint256,
+        uint256 _auctionPivotPrice
+    )
+        external
+        view
+    {
+        // Generically, require pivot price to be between 0.5 and 5
+        require(_auctionPivotPrice > priceDenominator.div(2) && _auctionPivotPrice < priceDenominator.mul(5));
```

Comment:
Can you put each check on its own line? I'd also prefer that constants / magic numbers are defined at the top of the file and referenced.

Reply:
How so? I think a comment right above the `require` is plenty enough. No need to have any extra variables or anything.

Reply:
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Also, if our plan might be to pass this in the constructor, makes sense to store in a variable.

My other comment was to do the following:

```
require(
   _auctionPivotPrice > priceDenominator.div(2) && 
  _auctionPivotPrice < priceDenominator.mul(5)
);
```

Also, let's add a require message?

Reply:
They should be on their own lines with proper require messages. The comment is fine